spec/api-app-spec.js

```diff
@@ -425,6 +425,52 @@ describe('app module', () => {
       w.loadURL('about:blank')
       w.webContents.executeJavaScript(`require('electron').remote.getGlobal('test')`)
     })
+
+    it('should emit remote-get-builtin event when remote.getBuiltin() is invoked', (done) => {
+      app.once('remote-get-builtin', (event, webContents, moduleName) => {
+        expect(webContents).to.equal(w.webContents)
+        expect(moduleName).to.equal('app')
+        done()
+      })
+      w = new BrowserWindow({
+        show: false,
+        webPreferences: {
+          nodeIntegration: true
+        }
+      })
+      w.loadURL('about:blank')
+      w.webContents.executeJavaScript(`require('electron').remote.app`)
```

Comment:
do you need to wait for the `loadURL` to complete?

Reply:
added